File: dev-docs/snippets/collections/add-document.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Add a document to a collection.
export async function addDocument(
  client: JsBaoClient,
  collectionId: string,
  documentId: string,
) {
  // #region example
  const entry = await client.collections.addDocument(collectionId, documentId);
  // #endregion example
  return entry;
}

File: dev-docs/snippets/collections/add-document.swift
import JsBaoClient

// Add a document to a collection. Returns a `CollectionDocumentInfo`.
func addDocument(
  client: JsBaoClient,
  collectionId: String,
  documentId: String
) async throws {
  // #region example
  let entry = try await client.collections.addDocument(
    collectionId: collectionId,
    documentId: documentId
  )
  // #endregion example
  _ = entry
}
